src/lumigo.ts

```diff
@@ -253,6 +287,9 @@ export class Lumigo {
   }
 
   public traceEverything(root: App | Stack, props: LumigoTraceProps = DEFAULT_LUMIGO_TRACE_PROPS) {
+    // For the people not using TypeScript, we need to perform some basic type validation
+    assertExpectedConstructType('traceEverything', 'App', root, (c) => c instanceof App);
```

Comment:
The signature also allows `Stack` as type of `root` but the validation checks only for `App`

Reply:
True